src/lib/webpack/run-webpack.js

```diff
@@ -26,23 +26,21 @@ const devBuild = async (env, onprogress) => {
 
 	let compiler = webpack(config);
 	return await new Promise((resolve, reject) => {
-		let first = true;
+
 		compiler.plugin('done', stats => {
-			if (first) {
```

Comment:
Removed this since it is better to clear off the console when there is an error and show the network interface when build is successful (every time)